src/Async/index.js

```diff
@@ -323,6 +323,42 @@ function Async(fn) {
     }
   }
 
+  function applyTo(method) {
+    return function (liftedApply) {
+      if (!isSameType(Async, liftedApply)) {
+        throw new TypeError(`Async.${method}: Async required`)
+      }
+
+      // liftedApply is Async[A => B]
+      return Async(function (reject, resolve) {
+        let resolvedA = null
+        let resolvedF = null
+        let cancelled = false
+
+        const rejectOnce = once(reject)
+
+        // todo: not sure if this is necessary
```

Comment:
Yep, you are correct!
This is an old artifact. You can remove it from your `applyTo`. Leave it in `ap` for now and I will make an issue as that is already released to the world.